src/api.ts

```diff
@@ -2,7 +2,9 @@ import * as vscode from 'vscode';
 import { performAction } from './actions/client';
 import { ActionsController } from './actions/controller';
 import { ExecuteAction, SnippetAction, CommandAction, FileAction } from './actions/impl';
+import { getLogger } from './logger/logger';
 export * from "./actions/interfaces";
+import _ = require('lodash');
```

Comment:
import order:
- don't mix import and exports. (export after all import)
- import of 3rd party libraries should appear **before** import of relative paths modules.